SpatialGDK/Source/SpatialGDKEditor/Private/SchemaGenerator/Utils/DataTypeUtilities.cpp

```diff
@@ -27,6 +27,10 @@ FString GetEnumDataType(const UEnumProperty* EnumProperty)
 
 FString UnrealNameToSchemaName(const FString& UnrealName)
 {
+	if (UnrealName.IsValidIndex(0) && FChar::IsDigit(UnrealName[0]))
```

Comment:
I just thought of a fun edge-case (sorry).
What if you have _1MyActor?
I think that could maybe be an allowed class name and will not be caught by this.

Reply:
The fix for this would be to first do the alphanumeric sanitization, and then do this check.

Reply:
You are right. The leading underscore will actually be stripped by `UnrealNameToSchemaName` but `ClassName` hasn't been stripped yet. So I actually need to perform the stripping first in any case.

Reply:
Fixed